app/ui/lib/DialogOverlay.tsx

```diff
@@ -2,21 +2,9 @@
  * This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, you can obtain one at https://mozilla.org/MPL/2.0/.
- *  
+ *
  * Copyright Oxide Computer Company
  */
+import { classed } from '~/util/classed'
 
-.DialogOverlay {
-  /* background: hsla(0, 0%, 0%, 0.33); */
-  @apply fixed inset-0 z-10 overflow-auto bg-scrim;
-}
-
-.DialogContent {
-  @apply z-20;
-
-  width: 50vw;
-  margin: 10vh auto;
-  background: white;
-  padding: 2rem;
-  outline: none;
-}
```

Comment:
I had copied this stuff in from Reach dialog when we switched to Radix, even though almost all of it was being overridden.

https://github.com/reach/reach-ui/blob/43f450db7bcb25a743121fe31355f2294065a049/packages/dialog/styles.css#L18-L24